x-pack/plugins/secops/public/components/error_toast/index.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiGlobalToastList, Toast } from '@elastic/eui';
+import { noop } from 'lodash';
+import React from 'react';
+import uuid from 'uuid';
+
+interface Props {
+  toastLifeTimeMs?: number;
+}
+
+interface State {
+  toasts: Toast[];
+}
+
+let showErrorHandler = ({ id = uuid.v4(), title, message }: ShowErrorParams): void => noop();
```

Comment:
Optional - why not just using an action from redux and connect the toaster to it or fancier create an observable in the error link

Reply:
I'll see if I can do one or the other for this.